lang_qc/main.py

```diff
@@ -55,3 +56,4 @@ class Settings(BaseSettings):
     allow_methods=["*"],
     allow_headers=["*"],
 )
+app.add_middleware(GZipMiddleware)
```

Comment:
Do we have to ask explicitly for compression, or this will happen on all responses now?

Reply:
The middleware defaults to compressing > 500 bytes, and only if the client asks for suitable encoding: https://www.starlette.io/middleware/#gzipmiddleware

It should be a harmless addition.